### PR TITLE
update change miner owner address and the Addresses section of the markets config.toml

### PR DESCRIPTION
**Pull request by dayou5168**
> The above command will need to be run twice, once by the original owner, and then a second time by the new owner to finalize the transfer

I think this statement is based on our old docs version. now we add two commond to the docs
and this statement also does not accurately describe the operation.

and add update the Addresses section of the markets config.toml,Update this section as same  as  mining/sealing/proving node

### CODE REVIEW

**Comment by johnnymatthews** [comment]
Closes #1028.

**Comment by dayou5168** [comment]
Closes#1028

File: docs/mine/lotus/split-markets-miners.md [hunk 1]
```diff
@@ -482,3 +482,13 @@ As soon as the message is confirmed, clients will know to look for the node iden
 ```shell
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
+9. update the Addresses section of the markets config.toml,Update this section as same  as  mining/sealing/proving node
```
Comment (johnnymatthews) [inline docs/mine/lotus/split-markets-miners.md:485]:
```suggestion

9. Update the `Addresses` section of the markets `config.toml` to be the same as same the mining/sealing/proving node.
```

- Add a space after each code block.
- Capitalize the start of sentences.
- Surround variable names like `Addresses` in back ticks (`).
- Surround file names like `config.toml` in back ticks (`).
- End sentences with a period/full-stop (.).

File: docs/mine/lotus/split-markets-miners.md [hunk 2]
```diff
@@ -482,3 +482,13 @@ As soon as the message is confirmed, clients will know to look for the node iden
...
+> [Addresses]  
+  \#PreCommitControl = ["f00XX1"]  
+  \# CommitControl = ["f00XX2"]  
+    TerminateControl = []  
+  \#  DealPublishControl = ["f00XX3"] 
+    DisableOwnerFallback = false  
+    DisableWorkerFallback = false
```
Comment (johnnymatthews) [inline docs/mine/lotus/split-markets-miners.md:492]:
````suggestion

```toml
[Addresses]  
  \#PreCommitControl = ["f00XX1"]  
  \# CommitControl = ["f00XX2"]  
    TerminateControl = []  
  \#  DealPublishControl = ["f00XX3"] 
    DisableOwnerFallback = false  
    DisableWorkerFallback = false 
```
````

Code examples should be in code blocks.

File: docs/mine/lotus/split-markets-miners.md [hunk 3]
```diff
@@ -482,3 +482,13 @@ As soon as the message is confirmed, clients will know to look for the node iden
...
+9. update the Addresses section of the markets config.toml,Update this section as same  as  mining/sealing/proving node
+> [Addresses]  
+  \#PreCommitControl = ["f00XX1"]  
+  \# CommitControl = ["f00XX2"]  
+    TerminateControl = []  
+  \#  DealPublishControl = ["f00XX3"] 
+    DisableOwnerFallback = false  
+    DisableWorkerFallback = false 
+ 
+Normally, if you initialize the markets node from the config.toml used in the mining/sealing/proving node, the settings in this section will be the same as in the mining/sealing/proving node, but if you are already running a subsystem and have changed the configuration in this area midway through the mining/sealing/ proving node, please make the same changes in the markets node, otherwise the DealPublishControl configuration will not take effect and DealPublish costs will be deducted from the worker wallet.see [this issue](https://github.com/filecoin-project/lotus/issues/7168)
```
Comment (johnnymatthews) [inline docs/mine/lotus/split-markets-miners.md:494]:
```suggestion
Typically, if you initialize the markets node from the `config.toml` used in the mining/sealing/proving node, the settings in this section will be the same as in the mining/sealing/proving node. If you are already running a subsystem and have changed the configuration in this area, copy the changes to the markets node. If you do not copy the changes, the `DealPublishControl` configuration will not affect, and `DealPublish` costs will be deducted from the worker wallet.
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Made a few minor updates, mostly to grammar and formatting. Please take a look add accept the suggestions when you're ready.

**Review by dayou5168 — COMMENTED** [review]
changed

**Review by dayou5168 — COMMENTED** [review]
changed

**Pull request closed without merging**